src/ds/ds-device-common.cpp

```diff
@@ -71,6 +71,13 @@ namespace librealsense
             LOG_INFO("entering to update state, device disconnect is expected");
             command cmd(ds::DFU);
             cmd.param1 = 1;
+
+            // TODO: DFU issue - A workdaroud till HKR FW team will support DFU response
+            if (dynamic_cast<d500_device*>(_owner))
+            {
```

Comment:
If you use `cmd.require_response = false;` for D400 devices something is broken? or it works OK?

Reply:
I can test this tomorrow, since I have no other devices connected for now.